BrowserServicesKit/Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -694,6 +706,8 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
     open override func startTunnel(options: [String: NSObject]? = nil) async throws {
         Logger.networkProtection.log("🚀 Starting tunnel")
 
+        observeSettingChanges()
```

Comment:
@diegoreymendez this one is tricky, I experienced a lot of issues testing the VPN, having the XPC connection timing out after 3 (seconds? attempts? I don't know) and after a few tests I noticed that the `XPacketTunnelProvider` init took a lot of time, so I tried to move as much work as possible outside the inits. Not sure if this is correct, the VPN works as expected.